system/Common.php

```diff
@@ -435,13 +442,6 @@ function esc($data, string $context = 'html', ?string $encoding = null)
         if (is_string($data)) {
             $context = strtolower($context);
```

Comment:
It seems this line also should be moved up.

Reply:
Oh, right. Sorry. It should be OK now.